rpmio/rpmpgp.c

```diff
@@ -1117,6 +1117,29 @@ static int pgpVerifySelf(pgpDigParams key, pgpDigParams selfsig,
     return rc;
 }
 
+static int parseSubkeySig(const struct pgpPkt *pkt, uint8_t tag,
+			  pgpDigParams *params_p) {
+    pgpDigParams params = *params_p = NULL; /* assume failure */
+
+    if (pkt->tag != PGPTAG_SIGNATURE)
+	goto fail;
+
+    params = pgpDigParamsNew(tag);
+
+    if (pgpPrtSig(tag, pkt->body, pkt->blen, params))
+	goto fail;
+
+    if (params->sigtype != PGPSIGTYPE_SUBKEY_BINDING &&
+	params->sigtype != PGPSIGTYPE_SUBKEY_REVOKE)
+	goto fail;
```

Comment:
Here's another case of misleading indentation wrt if.

Reply:
Yeah, checking for such things manually isn't very productive use of anybody's time.
If you know of automated tools that can do such stuff, it would certainly be welcome. Bonus points for CI automation...